GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -160,6 +162,49 @@ void Renderer::renderCamera(Scene* scene, Camera* camera, int framebufferId,
     // bone/weight/joint and other assimp data, we will put general model conversion
     // on hold and do this kind of conversion fist
 
+    //  LOGI(" Render Camera Framebuffer %d ", framebufferId);
+
+    DirectionalLight *cameraLight = scene->getDirectionalLight();
+
+    bool renderShadow = cameraLight; // TODO reader from scene;
+
```

Comment:
What is the default behavior of getDirectionalLight()? Will it return NULL or an instance of DirectionalLight?